### PR TITLE
[no-release-notes] dolt constraints verify for new format

### PR DESCRIPTION
**Pull request by druvv**
Reimplements dolt constraints verify to use the sql path when outputting rows.

### CODE REVIEW

File: go/cmd/dolt/commands/cvcmds/verify_constraints.go [hunk 1]
```diff
@@ -109,51 +111,103 @@ func (cmd VerifyConstraintsCmd) Exec(ctx context.Context, commandStr string, arg
...
-					return commands.HandleVErrAndExitCode(errhand.BuildDError("Error outputting rows").AddCause(err).Build(), nil)
-				}
+			dErr := printViolationsForTable(ctx, tableName, tbl, eng)
+			if dErr != nil {
+				return commands.HandleVErrAndExitCode(dErr, nil)
+			}
+		}
+
+		if outputOnly {
+			err = dEnv.UpdateWorkingRoot(ctx, working)
```
Comment (druvv) [inline go/cmd/dolt/commands/cvcmds/verify_constraints.go:144]: Is there a good way for the engine to be initialized using a given working set? We want to temporarily add the violations if `--no-output` is set.

File: go/cmd/dolt/commands/cvcmds/verify_constraints.go [hunk 2]
```diff
@@ -53,7 +55,7 @@ func (cmd VerifyConstraintsCmd) Description() string {
 }
 
 func (cmd VerifyConstraintsCmd) GatedForNBF(nbf *types.NomsBinFormat) bool {
```
Comment (zachmu) [inline go/cmd/dolt/commands/cvcmds/verify_constraints.go:57]: Prefer just remove this function

File: go/cmd/dolt/commands/cvcmds/verify_constraints.go (hunk 1 above)
Comment (zachmu) [reply go/cmd/dolt/commands/cvcmds/verify_constraints.go:144]: DoltSession.UpdateWorkingSet?

**Review by zachmu — APPROVED** [review]
LGTM

Should avoid updating working set like that though

**Pull request merged**